app/src/main/java/org/p2p/wallet/bridge/repository/BridgeRemoteRepository.kt

```diff
@@ -0,0 +1,29 @@
+package org.p2p.wallet.bridge.repository
+
+import com.google.gson.Gson
+import java.net.URI
+import org.p2p.core.rpc.JsonRpc
+import org.p2p.core.rpc.RpcApi
+import org.p2p.wallet.bridge.api.mapper.BridgeServiceErrorMapper
+import org.p2p.wallet.bridge.model.BridgeResult
+
+private val BRIDGE_BASE_URL = URI.create("https://bridge-service.keyapp.org/")
```

Comment:
we have this base URL from BridgesServiceEnvironment please use this provided in BridgeRemoteRepository